src/get-started/wordcount-example.md

```diff
@@ -618,12 +740,13 @@ public static void main(String[] args) {
 ```
 
 ```py
-This feature is not yet available in the Beam SDK for Python.
+# This feature is not yet available in the Beam SDK for Python.
```

Comment:
I do not believe this to be true, but can be fixed in a separate change

@robertwb

Reply:
filed BEAM-2976